code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -281,4 +281,17 @@
 /obj/item/weapon/storage/box/syndie_kit/demolitions_super_heavy/New()
 	..()
 	new /obj/item/weapon/syndie/c4explosive/heavy/super_heavy(src)
-	new /obj/item/weapon/screwdriver(src)
\ No newline at end of file
+	new /obj/item/weapon/screwdriver(src)
+
+
+/obj/item/weapon/storage/secure/briefcase/rifle
+	name = "secure briefcase"
+
+/obj/item/weapon/storage/secure/briefcase/rifle/New()
+	..()
+	new /obj/item/sniper_rifle_part/barrel(src)
+	new /obj/item/sniper_rifle_part/stock(src)
+	new /obj/item/sniper_rifle_part/trigger_group(src)
+
+	for(var/i = 1 to 5)
+		new /obj/item/ammo_casing/a145(src)
```

Comment:
It actually ends up being one ammo-casing (small item) larger in size difference from a regular briefcase, to where if I did 1 to 4, it'd be identical in size.